pkg/image/api/validation/validation.go

```diff
@@ -75,10 +75,6 @@ func ValidateImageStream(stream *api.ImageStream) field.ErrorList {
 		result = append(result, field.Invalid(field.NewPath("metadata", "name"), stream.Name, fmt.Sprintf("'namespace/name' cannot be longer than %d characters", reference.NameTotalLengthMax)))
 	}
 
-	if stream.Spec.Tags == nil {
```

Comment:
Actually... defaulting happens in conversion, anyone who is passing a nil type to validation won't break, but this is internal only.  I'm not really that worried here.